cmd/server/server.go

```diff
@@ -40,7 +40,8 @@ import (
 	"github.com/woodpecker-ci/woodpecker/remote"
 	"github.com/woodpecker-ci/woodpecker/router"
 	"github.com/woodpecker-ci/woodpecker/router/middleware"
-	droneserver "github.com/woodpecker-ci/woodpecker/server"
+	woodpecker "github.com/woodpecker-ci/woodpecker/server"
```

Comment:
We should leave the import as `server` to match the way we import it in all other files.

Reply:
Hmm. I assumed this was imported as droneserver because it was in a server.go file. I'll have to give that a try.